### PR TITLE
List of small fixes/improvement

### PR DESCRIPTION
**Pull request by ArtificialOwl**
- display colored text for Local instance in point:list,
- log some error in case of exception during cron,
- delay between health status check changed to 7d (from 12h)
- filters all logs from nextcloud, included incremented numbers,

### CODE REVIEW

File: lib/Command/PointList.php [hunk 1]
```diff
@@ -171,7 +172,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 						($fresh) ? $point->getParent() : '',
 						$comment,
 						implode(',', $status),
-						$instance,
+						($instance === RemoteInstance::LOCAL) ? '<info>' . $instance . '</info>' : $instance,
```
Comment (artonge) [inline lib/Command/PointList.php:175]: Wouldn't it be better to do it in the front-end side ?

File: lib/Service/FilesService.php [hunk 2]
```diff
@@ -107,7 +107,7 @@ public function fillRestoringData(RestoringData $data, string $path): void {
 		if (!is_dir($data->getAbsolutePath() . rtrim($path, '/'))) {
 			if ($data->getType() === RestoringData::ROOT_DATA
 				&& !$this->configService->getAppValueBool(ConfigService::INCLUDE_LOGS)
-				&& $path === 'nextcloud.log') {
+				&& substr($path, 0, 13) === 'nextcloud.log') {
```
Comment (artonge) [inline lib/Service/FilesService.php:110]: Might not be compatible with custom log file

**Review by artonge — CHANGES_REQUESTED** [review]

File: lib/Command/PointList.php (hunk 1 above)
Comment (ArtificialOwl) [reply lib/Command/PointList.php:175]: it's an occ command :]

File: lib/Service/FilesService.php (hunk 2 above)
Comment (ArtificialOwl) [reply lib/Service/FilesService.php]: I now get value of `logfile`

**Comment by ArtificialOwl** [comment]
and rebased.

**Review by artonge — APPROVED** [review]

**Pull request merged**